.github/workflows/trivy-scan.yml

```diff
@@ -23,12 +23,14 @@ jobs:
                     packagePAT: ${{ secrets.GITHUB_TOKEN }}
                 run: ./gradlew build -x check -x test
 
+            -   name: Create lib directory if not exists
+                run: mkdir -p transaction-ballerina/lib
```

Comment:
Does this mean there won't be /lib for trx? If so, what's the reason for scanning?

Reply:
Since there are no any 3rd party dependencies as of now, there won't be `lib` directory as of now. But in order to track any vulnerabilities related to only for this module without breaking in future, we decided to do this change and make the action consistent across all the repos in ballerina-platform.